app/views/collections/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="collections-index">
+  <h1 class="collections-index-title brand-alt-h1">All Collections</h1>
+  <div class="collections-index-list">
+    <% @collections.each do |col| %>
+        <div class="collection">
+            <%= link_to collection_path(col.friendlier_id) do %>
+            <div class="collection-thumb">
+                <%= ThumbDisplay.new(
+                    col.leaf_representative, thumb_size: :collection_page, placeholder_img_url: asset_path("default_collection.svg")).display %>
+                </div>
+                <div class="collection-title-wrapper">
+                  <p class="collection-title"><%= col.title %></p>
+                </div>
```

Comment:
fix indentation on all this stuff to all line up, close tags at same indentation as open tags.

Reply:
oh actually I think the div on line 7 is _missing_ it's close tag maybe?

Reply:
Done!